sklearn/feature_extraction/text.py

```diff
@@ -971,32 +971,50 @@ def _make_int_array():
 class TfidfTransformer(BaseEstimator, TransformerMixin):
     """Transform a count matrix to a normalized tf or tf-idf representation
 
-    Tf means term-frequency while tf-idf means term-frequency times inverse
-    document-frequency. This is a common term weighting scheme in information
-    retrieval, that has also found good use in document classification.
+    Definitions:
 
-    The goal of using tf-idf instead of the raw frequencies of occurrence of a
-    token in a given document is to scale down the impact of tokens that occur
-    very frequently in a given corpus and that are hence empirically less
-    informative than features that occur in a small fraction of the training
-    corpus.
+    Term-frequency (tf) = the frequency of a term in a document.
+
+    Inverse-document-frequency (idf) = 1 / (# of docs a term appears in)
+
+    This is a common term weighting scheme in information
+    retrieval, also commonly used in document classification.
+
+    Using the tf-idf transformed frequencies allows the raw term counts of
+    a term occurring in a document to be weighted by the frequency of the
+    term occurring in other documents in the corpus. Thus, terms that occur in
+    many documents in a corpus will be down-weighted, and terms that occur in
+    few documents in a corpus will be up-weighted. Frequently occurring terms
+    are empirically less informative than features that occur in a small
+    fraction of the training corpus.
 
     The formula that is used to compute the tf-idf of term t is
-    tf-idf(d, t) = tf(t) * idf(d, t), and the idf is computed as
-    idf(d, t) = log [ n / df(d, t) ] + 1 (if ``smooth_idf=False``),
-    where n is the total number of documents and df(d, t) is the
-    document frequency; the document frequency is the number of documents d
-    that contain term t. The effect of adding "1" to the idf in the equation
-    above is that terms with zero idf, i.e., terms  that occur in all documents
+
+    tf-idf(d, t, D) = tf(t, d) * idf(t, D)
```

Comment:
Can you maybe reproduce the formula also in the TfidfVectorizer docs? Otherwise looks good :) I feel like the two should largely have the same docs.